### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by ghost** [adepierre/Botcraft#35] [Q&A] [answered]
Will you have a discord server?

**Comment by adepierre** [comment]
I'm currently thinking about the possible solutions to make Botcraft more accessible. Opening a discord server is actually one of them.

**Comment by maxsupermanhd** [comment]
Poke? Any news about possible Discord server?

**Reply by adepierre** [reply]
It's actually in construction, but I need to wait for something personal before seeing if I release it or not.

**Reply by amoraschi** [reply]
I know this is old, but is there any updates on this?

**Reply by adepierre** [reply]
I'm not a big discord user so I have to admit I kinda forgot. I'll try to see if I can go on with that.

**Comment by adepierre** [comment, marked as answer]
The community discord server is now online. Anyone wanting to join can use [this invitation link](https://discord.gg/wECVsTbjA9).
